src/cf.data.pre

```diff
@@ -51,9 +51,11 @@ COMMENT_START
 
 	Units accepted by Squid are:
 		bytes - byte
-		KB - Kilobyte (1024 bytes)
-		MB - Megabyte
-		GB - Gigabyte
+		KB - Kilobyte (2^10, 1024 bytes)
+		MB - Megabyte (2^20, 1'048'576 bytes)
+		GB - Gigabyte (2^30, 1'073'741'824 bytes)
+	Squid does not use their respective IEEE names KiB, MiB, and GiB
+	to preserve backwards compatibility
```

Comment:
Using "their respective IEEE names KiB, MiB, and GiB" (in addition to current units) would not break backwards compatibility.

Also, I would not mention IEEE in this context. There are too many standards defining these names. We do not have to mention one since we are not supporting these names at all. FWIW, [Wikipedia](https://en.wikipedia.org/wiki/Binary_prefix) only mentions IEEE in "obsolete standards" context.

```suggestion
	Squid does not yet support KiB, MiB, and GiB unit names.
```